draft-ietf-spice-sd-cwt.md

```diff
@@ -196,122 +279,78 @@ sd-protected = {
    * key => any
 }
 
-unprotected-presentation = {
-   &(sd_kbt: TBD2) ^ => bstr .cbor kbt-cwt,
-   ? &(sd_claims: TBD1) ^ => bstr .cbor [ + bstr .cbor salted ],
+unprotected-issued = {
+   &(sd_claims: TBD1) ^ => bstr .cbor salted-array,
    * key => any
 }
 
 sd-payload = {
     ; standard claims
       &(iss: 1) ^ => tstr, ; "https://issuer.example"
       &(sub: 2) ^ => tstr, ; "https://device.example"
-      &(aud: 3) ^ => tstr, ; "https://verifier.example"
+    ? &(aud: 3) ^ => tstr, ; "https://verifier.example/app"
     ? &(exp: 4) ^ => int,  ; 1883000000
```

Comment:
Why is "exp" optional?  It seems like it should be required.

Reply:
RFC8392 is silent about if exp is required in a CWT. It refers to the definition in [Section 4.1.4 of RFC7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4) which says `exp` is optional in a JWT.

Reply:
For what it's worth, *all* claims are optional in JWTs because they are expected to be profiled for particular use cases.  In our case, we may want to define the minimum required claims.

I thought that the "?" syntax above made "exp" optional.  Am I correct?  And similarly, the change above makes "aud" optional.  Is that what we want to do?

Reply:
I have a legitimate use case for "aud" to be optional (use in MLS), but the "aud" would be required in the Key Binding Token.

For "exp" I cannot think of a use case where we want a never expiring SD-CWT, but I was hesitant to change the optionality and break someone else's use case.

Reply:
I created Issue #14 for this. I propose we resolve in this PR and incorporate the result of this conclusion in another PR.